website/allowed-subcategories.txt

```diff
@@ -103,6 +103,7 @@ SimpleDB
 Step Function (SFN)
 Storage Gateway
 Transfer
+Transit Gateway Network Manager
```

Comment:
I went with this name as that's name Amazon use: https://aws.amazon.com/transit-gateway/network-manager/.